src/app/molecules/message/Message.jsx

```diff
@@ -228,10 +233,11 @@ MessageOptions.propTypes = {
 
 function Message({
   avatar, header, reply, content, editContent, reactions, options,
+  msgType,
 }) {
   const msgClass = header === null ? ' message--content-only' : ' message--full';
   return (
-    <div className={`message${msgClass}`}>
+    <div className={`message${msgClass} message--type-${msgType.replace('.', '-')}`}>
```

Comment:
Please tell if you have a preferred style for dynamic class names based on data.

Here I'm transforming the event type (`m.text`) to a form usable as a class name part (`m-text`).

Reply:
I personally do not prefer dynamic classes, because there can be uncertain class name collisions.

Reply:
Which pattern would you like to see used here? I can think of a few, some bad, some not great, some good, with different requirements for maintaining.

Note that we probably need to handle `m.notice` too, to completely support the spec, though at the time both `m.text` and `m.notice` will have no additional styles.

Reply:
What I can think of is appending a new class to myClass if msgType is <abc>